lib/router/mbaas.js

```diff
@@ -11,27 +12,32 @@ function initRouter(mediator) {
     var params = req.body; // params.userId params.password
```

Comment:
It should expect `userId` and specify that as part of the documentation for the module.